docker_test_tools/environment.py

```diff
@@ -229,7 +229,10 @@ def is_container_ready(self, name):
 
         :param str name: container name as it appears in the docker compose file.
         """
-        status_output = self.inspect_container(name)['State']
+        try:
+            status_output = self.inspect_container(name)['State']
+        except RuntimeError:
```

Comment:
RuntimeError is the exception that returned if the container doesn't exist?

Reply:
Yes: https://github.com/Stratoscale/docker-test-tools/blob/master/docker_test_tools/environment.py#L381